tools/starman-development.psgi

```diff
@@ -55,14 +55,18 @@ Log::Log4perl::init(\$LedgerSMB::Sysconfig::log4perl_config);
 
 builder {
 
-    enable 'InteractiveDebugger';
+#    enable 'InteractiveDebugger';
```

Comment:
Maybe we need to check for its availability and/or it being configured in 'ledgersmb.conf'? That feels much better than enabling/disabling code by putting it in comments.

Reply:
Agreed, although at this point, those are mostly to help others develop and are not as standardize as the application.

Reply:
Agreed. So these could be settings in Sysconfig's Debug section. I still feel like commenting out code "because it is for debugging" is a workaround for want of a real solution. Using config feels much more solid. 

Note that I'm not arguing that this code be integrated with the regular starman.psgi .